ocs_ci/ocs/resources/pod.py

```diff
@@ -247,16 +249,25 @@ def run_io(
 
 # Helper functions for Pods
 
-def get_all_pods(namespace=None):
+def get_all_pods(namespace=None, selector=None):
     """
     Get all pods in a namespace.
-    If namespace is None - get all pods
+
+    Args:
+        namespace (str): Name of the namespace
+        If namespace is None - get all pods
```

Comment:
If the comments are spread over multiple line, leave 4 spaces in next line

eg:
```
Args:
        namespace (str): Name of the namespace
            If namespace is None - get all pods
```

Reply:
done